autopush/tests/test_web_validation.py

```diff
@@ -158,13 +158,13 @@ def _make_fut(self):
         schema.context["log"] = Mock()
         return schema
 
-    def _make_test_data(self, headers=None, body="", path_args=None,
-                        path_kwargs=None, arguments=None):
+    def _make_test_data(self, headers=None, body="", args=None, kwargs=None,
+                        arguments=None):
```

Comment:
Punting on this for now, I'll revisit it later. cyclone apparently sets up path_args/kwargs for use by the prepare method, so I guess there's a small case to keep the naming, but I wouldn't mind shortening. the websocket handlers still use regular args (though they don't validate), should probably switch those over